learn_ai/assignments/text_classification_assignment_1.ipynb

```diff
@@ -0,0 +1,1808 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    qdrant_client = QdrantClient(":memory:")
</pre><p>This is just Numpy for reference — and not Qdrant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/FuturePathAI/Learn-AI-Engineering/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>